{{ cookiecutter.app_name }}/CHANGELOG

```diff
@@ -0,0 +1,5 @@
+# {{ cookiecutter.formal_name }} Release Notes
+
+## 0.0.1 ({% now 'local', '%d %b %Y' %})
+
+* Initial relase
```

Comment:
Furthermore, though, `rpmbuild` complains since this doesn't match the format it's expecting.
```
Docker| error: bad date in %changelog: Initial relase
Docker| warning: source_date_epoch_from_changelog set but %changelog is missing
```

I think this is the format it's looking for:
```
* Day-of-Week Month Day Year Name Surname <email> - Version-Release
- Change 1
- Change 2
```

Reply:
Huh - I saw the warning, but it wasn't causing a hard error for me...

Reply:
It's logged as `error` but it doesn't stop the build for me. It seemingly controls the setting of `SOURCE_DATE_EPOCH`....although, I'm not sure the importance of this downstream.

Reply:
I think I can live with this for now; if RPM is going to require a specific format for changelogs, then we're probably going to need a lot more customisation - because if RPM needs a specific format, you can bet Debian is going to need a *different* specific format (but it's just not getting flagged during build time). If the packages functionally *work*, then that's a major step forward.